### DISCUSSION TITLE
Feedback on some functionality

### DISCUSSION THREAD

**Discussion by Brick-Top** [rgriebl/brickstore#525] [Ideas]
Good evening All, first, thank you very much for keeping this program updated. It is awesome and the work you are doing is great!

I just wanted to give some feedback on a few new updates which has hindered my team, to see if we could get options for them.

1. The filter bar is now hidden behind a button. We almost always need to use the bar, so hiding it behind a button just requires that we have to click the button 100% of the time. This is not bothersome, but just something that added more work for us, without much QOL improvement.
2. The new tabs dropdown. We regularly work with 5-6 files at the same time, having to move between files. This new tabs dropdown makes it much harder for us to quickly move between files, and slows everything down. We have had to revert to previous updates. Could there perhaps be another UI option to show this? Or maybe I missed it, sorry!
3. For some reason, the new update opens new files in new windows, instead of new tabs. This might be on my side too, I did not have time to troubleshoot, but wanted to share. 

Again, thank you for updating this program, it works amazing compared to the old Brickstore!!

**Comment by rgriebl** [comment]
> 1. The filter bar is now hidden behind a button. We almost always need to use the bar, so hiding it behind a button just requires that we have to click the button 100% of the time. This is not bothersome, but just something that added more work for us, without much QOL improvement.

Short term, `Ctrl+F` is a much quicker way than using your mouse


> 2\. The new tabs dropdown. We regularly work with 5-6 files at the same time, having to move between files. This new tabs dropdown makes it much harder for us to quickly move between files, and slows everything down. We have had to revert to previous updates. Could there perhaps be another UI option to show this? Or maybe I missed it, sorry!

Use the new dock widgets! In your case the `Open Documents` one. You can drag them around by their title bar to re-arrange and even stack them


> 3\. For some reason, the new update opens new files in new windows, instead of new tabs. This might be on my side too, I did not have time to troubleshoot, but wanted to share.

You need to give me more information here: which BrickStore version exactly? which OS platform? how are you opening these files? if from Windows explorer: how many at the same time?

**Comment by Brick-Top** [comment]
Thank you very much for your reply, and letting me know that Ctrl f opens the same dialog. I am sure you have future plans for the bar, but I thought that if that was to save a full bar of empty space (the bar where the filter used to sit now only shows price and lots/quantity of items) without much other use, it would be preferred to have the search bar open there.

I will test the dock widgets today. I must admit I uninstalled and reinstalled an old version as soon as I saw this change, as we could not afford to waste time on this. 

This was happening on windows, latest version. Windows 10, opening with ctrl+o worked to keep it in the same document, but from explorer any file clicked would open a new window. MAC works fine.

**Comment by ZZJHONS** [comment]
I still have mixed feelings with not having tabs on top, but I could't use much BS as much lately.

**Comment by Brick-Top** [comment]
One more piece of feedback which has been annoying me for some time. I can't use 'home' or 'end' keys on the keyboard to go to the beginning or end of the document. It goes to the beginning or end of whatever listing I have selected (goes horizontal instead of vertical).

**Comment by Brick-Top** [comment]
Any possibility that we could have an option to go back to tabs instead of the selector? Or is it in already and missed it?

Thanks!

**Comment by ZZJHONS** [comment]
Browsers use tabs, Acrobat uses tabs, Excel uses tabs, even Windows is updating the explorer to have tabs... After a year I'm still not comfortable using the new selector or widget.

**Comment by Brick-Top** [comment]
Further feedback on new implementations. I strongly dislike the way consolidate lots is now shown. Previously it was one screen and done faster, it is not in two different screens for some reason.

Not trying to beat you down, amazing program, just thought you'd like feedback.
